LibraryApp/src/redux/auth/actions.js

```diff
@@ -24,14 +25,32 @@ const privateActionCreators = {
       type: actions.LOGIN_FAILURE,
       payload: problem
     });
+  },
+  initWithStoredUser: () => dispatch => {
+    dispatch({
+      type: actions.INIT_STORED_USER
+    });
+    dispatch(
+      NavigationActions.navigate({
+        routeName: Routes.Library
+      })
+    );
   }
 };
 
 const actionCreators = {
+  initRemembered: () => dispatch => {
```

Comment:
This should be `initialLoading` and here you should see if the you have a token stored in localStorage (should call a service function that returns it).

Then you have to create a condition, if there are tokens, go to Library, else go to Login.

Take a look to YummIQ